src/BrowscapUpdater.php

```diff
@@ -154,6 +154,20 @@ public function getLogger()
     }
 
     /**
+     * Sets the Connection Timeout
+     *
+     * @param $connectTimeout
```

Comment:
Type is missing from docblock, I believe it should be `@param int $connectTimeout`